File: routes/python-to-go/certification/artifacts/holdout/route_harness.go
package main

import (
    "encoding/base64"
    "fmt"
)

func main() {
    actual0 := clamp(20, 10)
    if actual0 != 10 { panic("case 0") }
    fmt.Printf("ELMOS_OBSERVATION\t0\tb64\t%s\n", base64.StdEncoding.EncodeToString([]byte(fmt.Sprint(actual0))))
    actual1 := clamp(-2, 10)
    if actual1 != 0 { panic("case 1") }
    fmt.Printf("ELMOS_OBSERVATION\t1\tb64\t%s\n", base64.StdEncoding.EncodeToString([]byte(fmt.Sprint(actual1))))
    actual2 := clamp(7, 10)
    if actual2 != 7 { panic("case 2") }
    fmt.Printf("ELMOS_OBSERVATION\t2\tb64\t%s\n", base64.StdEncoding.EncodeToString([]byte(fmt.Sprint(actual2))))
}

File: routes/python-to-java/certification/artifacts/holdout/RouteHarness.java
public final class RouteHarness {
    public static void main(String[] args) {
        var actual0 = Migrated.clamp(20, 10);
        if (actual0 != 10) throw new AssertionError("case 0");
        System.out.println("ELMOS_OBSERVATION\t0\tb64\t" + java.util.Base64.getEncoder().encodeToString(String.valueOf(actual0).getBytes(java.nio.charset.StandardCharsets.UTF_8)));
        var actual1 = Migrated.clamp(-2, 10);
        if (actual1 != 0) throw new AssertionError("case 1");
        System.out.println("ELMOS_OBSERVATION\t1\tb64\t" + java.util.Base64.getEncoder().encodeToString(String.valueOf(actual1).getBytes(java.nio.charset.StandardCharsets.UTF_8)));
        var actual2 = Migrated.clamp(7, 10);
        if (actual2 != 7) throw new AssertionError("case 2");
        System.out.println("ELMOS_OBSERVATION\t2\tb64\t" + java.util.Base64.getEncoder().encodeToString(String.valueOf(actual2).getBytes(java.nio.charset.StandardCharsets.UTF_8)));
    }
}
